cmd/fortifyExecuteScan.go

```diff
@@ -605,7 +605,7 @@ func autoresolveMavenClasspath(config fortifyExecuteScanOptions, file string, ut
 	}
 	_, err := maven.Execute(&executeOptions, utils)
 	if err != nil {
-		log.Entry().WithError(err).Warn("failed to determine classpath using Maven")
+		log.Entry().WithError(err).Fatal("failed to determine classpath using Maven")
```

Comment:
fatal will abort the execution.
We agreed to bubble up errors in such a case thus we should return the error ...

Reply:
Right, corrected coding accordingly, thx.